### PR TITLE
Move the thoth-station prod apps to smaug

### PR DESCRIPTION
**Pull request by harshad16**
Move the thoth-station prod apps to smaug
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

Related-to: https://github.com/thoth-station/thoth-application/issues/1971

Details:
1. moving all the thoth-station workloads from zero to Smaug
2. As pre-requisites like pvc and ceph is not available, the prod app is currently not added to [kustomization.yaml](https://github.com/harshad16/argocd-apps/blob/26ddf1ff11f64c0290fa2253830d158592a04ec9/envs/moc/smaug/thoth/kustomization.yaml)
3. Bots that are not dependent are switched on.

### CODE REVIEW

**Comment by harshad16** [comment]
/hold
wait for #204

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Review by tumido — CHANGES_REQUESTED** [review]
as @HumairAK noted here https://github.com/operate-first/argocd-apps/pull/204#issuecomment-921224874

Please add the smaug cluster to your project:
https://github.com/operate-first/argocd-apps/blob/b3165a430accd2655131957889f96c13286972fe/projects/thoth.yaml#L8-L10

**Comment by harshad16** [comment]
/unhold
Ready for Review

File: envs/moc/smaug/thoth/prod/kustomization.yaml [hunk 1]
```diff
@@ -26,14 +24,11 @@ resources:
...
 - prod-thoth-package-extract.yaml
 - prod-thoth-package-releases.yaml
 - prod-thoth-package-update.yaml
 - prod-thoth-purge-job.yaml
 - prod-thoth-revsolver.yaml
 - prod-thoth-security-indicators.yaml
-- prod-thoth-sefkhet-abwy.yaml
 - prod-thoth-solver.yaml
-- prod-thoth-triage-party.yaml
 - prod-thoth-user-api.yaml
```
Comment (Gregory-Pereira) [inline envs/moc/smaug/thoth/prod/kustomization.yaml:34]: Both the file was changed and these components were removed?

**Review by Gregory-Pereira — COMMENTED** [review]
looks good, just that one question on the review.

File: envs/moc/smaug/thoth/prod/kustomization.yaml (hunk 1 above)
Comment (harshad16) [reply envs/moc/smaug/thoth/prod/kustomization.yaml:34]: This file is no longer part of the **prod** directory, it is moved to the **bots** directory.

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**